lms/templates/courseware/course_about.html

```diff
@@ -228,15 +227,15 @@
               </a>
             % else:
               <%
-                site_domain = microsite.get_value('site_domain', settings.SITE_NAME)
-                platform_name = microsite.get_value('platform_name', settings.PLATFORM_NAME)
+                site_domain = static.get_value('site_domain', settings.SITE_NAME)
+                platform_name = static.get_value('platform_name', settings.PLATFORM_NAME)
```

Comment:
we already have `get_platform_name` so we can use here like this `platform_name=static.get_platform_name()`